.ci/install_runtime.sh

```diff
@@ -110,7 +126,15 @@ case "${KATA_HYPERVISOR}" in
 		enable_hypervisor_config "${PKGDEFAULTSDIR}/configuration-fc.toml"
 		;;
 	"qemu")
-			enable_hypervisor_config "${PKGDEFAULTSDIR}/configuration-qemu.toml"
+		case "$TEE_TYPE" in
+			"tdx")
+				enable_hypervisor_config "${PKGDEFAULTSDIR}/configuration-qemu-tdx.toml"
```

Comment:
I'm also doing something similar as part of https://github.com/kata-containers/tests/pull/5065

Maybe here is the right place to do that, and you'll have to adapt what I'm doing there, but this looks fine!

Reply:
Also, we'll need to expand this for the Cloud Hypervisor case.